decidim-core/db/migrate/20240401192628_change_name_on_decidim_organizations.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ChangeNameOnDecidimOrganizations < ActiveRecord::Migration[6.1]
+  class Decidim::MigrationOrganization < ApplicationRecord
+    self.table_name = :decidim_organizations
+  end
+
+  def up
+    rename_column :decidim_organizations, :name, :old_name
+    add_column :decidim_organizations, :name, :jsonb, null: false, default: {}
+
+    Decidim::MigrationOrganization.reset_column_information
+
+    Decidim::MigrationOrganization.find_each do |organization|
+      organization.update(name: { organization.default_locale => organization.old_name })
+    end
+    remove_column :decidim_organizations, :old_name
+  end
+
+  def down
+    rename_column :decidim_organizations, :name, :old_name
+    add_column :decidim_organizations, :name, :string, null: false, default: ""
+
+    Decidim::MigrationOrganization.reset_column_information
+
+    Decidim::MigrationOrganization.find_each do |organization|
+      organization.update(name: organization.old_name[organization.default_locale])
+    end
```

Comment:
```suggestion
  class Organization < ApplicationRecord
    self.table_name = :decidim_organizations
  end

  def up
    rename_column :decidim_organizations, :name, :old_name
    add_column :decidim_organizations, :name, :jsonb, null: false, default: {}

    Organization.reset_column_information

    Organization.find_each do |organization|
      organization.update(name: { organization.default_locale => organization.old_name })
    end
    remove_column :decidim_organizations, :old_name
  end

  def down
    rename_column :decidim_organizations, :name, :old_name
    add_column :decidim_organizations, :name, :string, null: false, default: ""

    Organization.reset_column_information

    Organization.find_each do |organization|
      organization.update(name: organization.old_name[organization.default_locale])
    end
```

Reply:
I've just checked out how we've done it in other migrations and I saw that we always do it like `Organization` without the `Decidim` namespace nor the `Migration` prefix.